tests/app/questionnaire/test_rules.py

```diff
@@ -97,6 +97,37 @@ def test_evaluate_rule_equals_with_number(self):
         self.assertFalse(evaluate_rule(when, 2))
         self.assertTrue(evaluate_rule(when, 0))
 
+    def test_evaluate_rule_equals_with_string(self):
+        when = {"value": "answervalue", "condition": "equals"}
+
+        self.assertTrue(evaluate_rule(when, "answerValue"))
+        self.assertTrue(evaluate_rule(when, "answervalue"))
+        self.assertFalse(evaluate_rule(when, "answer-value"))
```

Comment:
I find these hyphen comparison a bit odd, any reason for doing it?

Reply:
I added them here to test the negation 👍

Reply:
I understood why. Belt and braces programming

Reply:
I think copy and paste programming is more apt 😉